bootloader/bootloader.go

```diff
@@ -259,6 +262,15 @@ type RebootBootloader interface {
 	GetRebootArguments() (string, error)
 }
 
+// UefiBootloader provides data for setting EFI boot variables.
+type UefiBootloader interface {
+	Bootloader
+
+	// EfiLoadOptionParameters returns the data which may be used to construct
+	// an EFI load option.
+	EfiLoadOptionParameters(updatedAssets []string) (description string, assetPath string, optionalData []byte, err error)
```

Comment:
I find the `Load` part of the name a bit confusing as it sounds like it is taking the data from the system. Maybe `CreateEfiEntryParameters` or something like that?

Reply:
"load option" comes from the UEFI specs.

Reply:
I know, but still it sounds confusing :slightly_smiling_face: . Maybe `CreateEfiLoadOptionParameters`?

Reply:
but is returning parameters not creating the option.  You could switch things around:

ParametersForEfiLoadOption  or EfiParametersForLoadOption